fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -205,6 +205,19 @@ void applyMetadataMutations(
 				}
 
 				// Storage Team ID to Storage Server List
+				if (!initialCommit) {
+					txnStateStore->set(KeyValueRef(m.param1, m.param2));
+				}
+			} else if (m.param1.startsWith(splitKeyToStorageTeamIDPrefix)) {
```

Comment:
This is not needed if we use existing `keyServer`, `serverKey`, and `storageTeamIdKey` mappings.